web-ng/src/styles.scss

```diff
@@ -42,6 +42,19 @@ button .mat-icon + label {
   color: mat-color($foreground, icon) !important;
 }
 
+snack-bar-container.success {
+  background-color: mat-color($primary);
+}
+
+snack-bar-container.error {
+  background-color: mat-color($warn);
+}
+
+snack-bar-container.notification {
+  color: white;
```

Comment:
@parulraheja98 Is there a `mat-color` we can use here instead? Perhaps submitted in a followup PR? :)